src/test/java/ru/practicum/shareit/booking/BookingServiceTest.java

```diff
@@ -96,15 +156,34 @@ void getBookingById() {
     }
 
     @Test
-    void getBookingsByUser() {
+    void getBookingsByUserNotFound() {
+        bookingDto.getBooker().setId(99);
+        when(bookingRepository.findByBookerIdOrderByStartDesc(bookerId, page))
+                .thenReturn(new ArrayList<>());
+        assertThatThrownBy(() -> {
+            List<BookingDto> bookingsToGet = bookingService.getBookingsByUser(bookerId, "ALL", page);
+        }).isInstanceOf(NoSuchBooking.class).hasMessage("Bookings were not found");
+    }
+
+    @Test
+    void getBookingsByUserInvalidState() {
+        when(bookingRepository.findByBookerIdOrderByStartDesc(bookerId, page))
+                .thenReturn(new ArrayList<>());
+        assertThatThrownBy(() -> {
+            List<BookingDto> bookingsToGet = bookingService.getBookingsByUser(bookerId, "UNKNOWN", page);
```

Comment:
```suggestion
             bookingService.getBookingsByUser(bookerId, "UNKNOWN", page);
```
переменная не используется, соответсвенно создавать ее не нужно